src/json-rpc/types.ts

```diff
@@ -115,12 +115,12 @@ export interface TrackPackageRPCParams {
 export interface TrackPackageDTO {
   shipment: {
     carrierCode: string;
-    carrierAccountID: string;
-    shipmentID?: string;
+    carrierAccountId: string;
```

Comment:
Based on the discussions in [slack](https://auctane.slack.com/archives/C01G1SBEC1W/p1624475161002500)